src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -618,7 +618,10 @@ private BoundStatement BindDeclarationStatementParts(LocalDeclarationStatementSy
             {
                 kind = LocalDeclarationKind.Constant;
             }
-
+            else if (node.UsingKeyword != default)
+            {
+                kind = LocalDeclarationKind.UsingVariable;
+            }
```

Comment:
this looks to be going beyond just a parsing change (as the title of the PR implies).